src/parser.ts

```diff
@@ -927,8 +927,8 @@ export class Parser extends DiagnosticEmitter {
       if(!expression) return null;
       expression = this.maybeParseCallExpression(tn, expression);
       var isCallableExpression = expression.kind === NodeKind.CALL;
-      var localCallableVarDeclaration;
-      var localCallableIdentifier;
+      var localCallableVarDeclaration: VariableDeclaration | undefined;
+      var localCallableIdentifier: IdentifierExpression | undefined;
```

Comment:
AssemblyScript doesn't support `undefined`. Only `null`. And due to compiler support self-bootstrapping you should never use `undefined` even in compiler's codebase.